SPEC/SWARM.md

```diff
@@ -30,6 +31,29 @@ ipfs.swarm.addrs(function (err, peerInfos) {
 
 A great source of [examples][] can be found in the tests for this API.
 
+#### `swarm.localAddrs`
+
+> List of known local addresses of each peer connected.
```

Comment:
Not quite correct, these are the addresses your node is listening on.

```suggestion
> List local addresses this node is listening on.
```

Reply:
Fixed.